tests/unit/components/video-box/VideoBox.spec.jsx

```diff
@@ -1,19 +1,27 @@
 import { screen, render } from '@testing-library/react'
-
 import VideoBox from '~/components/video-box/VideoBox'
 
-const props = {
-  video: 'video.mp4'
-}
-
 describe('VideoBox component', () => {
-  beforeEach(() => {
-    render(<VideoBox { ...props } />)
+  const props = {
+    video: 'video.mp4',
+    videoPreview: true
+  }
+
+  it('should render video preview when videoPreview is true', () => {
```

Comment:
It's not crucial here, but we can make the description clearer:
`should render video preview when a user doesn't provide a link to the video presentation`.
The same for the opposite scenario.

Reply:
I don't think it is necessary to check for two values. it is clear from the variable name that it is a video. if video !== null or empty string => execute action